gobblin-iceberg/src/main/java/org/apache/gobblin/iceberg/writer/IcebergMetadataWriter.java

```diff
@@ -148,6 +158,29 @@
   /* one of the fields in DataFile entry to describe the location URI of a data file with FS Scheme */
   private static final String ICEBERG_FILE_PATH_COLUMN = DataFile.FILE_PATH.name();
 
+  public static final String ICEBERG_COMPLETENESS_ENABLED = "iceberg.completeness.enabled";
+  private static final boolean DEFAULT_ICEBERG_COMPLETENESS = false;
+  private final boolean completenessEnabled;
+  public static final String ICEBERG_COMPLETENESS_WHITELIST = "iceberg.completeness.whitelist";
+  public static final String ICEBERG_COMPLETENESS_BLACKLIST = "iceberg.completeness.blacklist";
+  private final WhitelistBlacklist completenessWhitelistBlacklist;
+  public static final String COMPLETION_WATERMARK_KEY = "completion.watermark";
```

Comment:
"completionWatermark" instead of completion.watermark? At LI, we already have a precedence of using the former.